.github/workflows/test.yml

```
@@ -107,5 +107,15 @@ jobs:
         # because we have a yarn.lock and caches efficiently.
         uses: bahmutov/npm-install@v1
 
+      - name: 📥 Install Playwright
+        run: npx playwright install --with-deps
+
       - name: 👀 Run Integration Tests
         run: "yarn test:integration"
+
+      - name: 📤 Upload test results
+        if: always()
+        uses: actions/upload-artifact@v2
```

v3 is already released

```suggestion
        uses: actions/upload-artifact@v3
```

---

just following the playwright docs 😅

---

Just wanted to let you know 😉